File: OMSWeb/Models/Tracks/FireShutter.cs
namespace OMSWeb.Models.Tracks
{
    public class FireShutter
    {
        public int Id { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public string LogicalId { get; set; }
        public string Segments { get; set; }
        public int Status { get; set; }

    }


    public class FireShutterStatus : Entities.FireShutterStatusEntity
    {
    }
}

File: OMSWeb/frontend/packages/oms-track-monitor/src/TrackObjects/fireshutter/types/Fireshutter.ts
import { CanBeFocused } from "src/MapObjects/focus/types/CanBeFocused"

type Fireshutter = {
	id: number
	x: number
	y: number
	logicalId: string
	segments: string
	status: number
  fireDetect: number
  open:number
} & CanBeFocused

export { Fireshutter }
